sequence_processing_pipeline/tests/test_Pipeline.py

```diff
@@ -792,7 +806,458 @@ def test_get_project_info(self):
 
         # test sample-information-file generation.
         pipeline = Pipeline(self.good_config_file, self.good_run_id,
-                            self.good_sample_sheet_path,
+                            self.good_sample_sheet_path, None,
+                            self.good_output_file_path, self.good_qiita_id,
+                            None)
+
+        obs_proj_info = pipeline.get_project_info()
+        obs_project_names = []
+        for d in obs_proj_info:
+            obs_project_names.append(d['project_name'])
+
+        self.assertEqual(sorted(obs_project_names), sorted(exp_project_names))
+
+        for exp_d in exp_proj_info:
+            for obs_d in obs_proj_info:
+                if obs_d['project_name'] == exp_d['project_name']:
+                    self.assertDictEqual(obs_d, exp_d)
+                    break
+
+
+class TestAmpliconPipeline(unittest.TestCase):
+    def setUp(self):
+        package_root = abspath('./sequence_processing_pipeline')
+        self.path = partial(join, package_root, 'tests', 'data')
+        self.good_config_file = join(package_root, 'configuration.json')
+        self.bad_config_file = self.path('bad_configuration.json')
+        self.invalid_config_file = 'does/not/exist/configuration.json'
+        self.good_run_id = '211021_A00000_0000_SAMPLE'
+        # good_qiita_id is randomly-generated and does not match any known
```

Comment:
AFAIK there is no "bad_qiita_id" so should this be just `qiita_id`?

Reply:
Ok.